### PR TITLE
Separate node and browser builds

### PR DESCRIPTION
**Pull request by diegomura**
- Create separate entry points for browser and node builds
- Separate lib real logic from entry points as a function that takes data and trie info 
- Make package json point to appropriate build for each env

@devongovett what do you think? I think I can change this approach to have one single entry point for both envs if you like, but probably going to require some conditionals and extra config on the rollup side.

I'll ultimately need your help for publishing 😄

### CODE REVIEW

**Comment by diegomura** [comment]
@devongovett kind reminder of his issue 😊

Would love to know what you think, in addition of your permissions to publish in case you agree with the changes.

File: generate.js [hunk 1]
```diff
@@ -101,3 +101,7 @@ fs.writeFileSync('./data.json', JSON.stringify({
   scripts: Object.keys(scripts),
   eaw: Object.keys(eaws)
 }));
+
+// Trie is serialized suboptimally as JSON so it can be loaded via require,
+// allowing unicode-properties to work in the browser
+fs.writeFileSync('./trie.json', JSON.stringify(trie.toBuffer()));
```
Comment (devongovett) [inline generate.js]: Maybe encoding it as base64 would be smaller?

Comment (diegomura) [reply generate.js]: Going to test it and check if that's true 😄 
Thanks!

Comment (diegomura) [reply generate.js]: Yes. Good call @devongovett !

Base64 encoded trie: 11,279 bytes
Trie raw file: 8,451 bytes.

Considering they are so close, do you think we should leave them separate anyway, or use encoded base64 for both builds and get back to having only one entry point to the lib?

**Comment by diegomura** [comment]
Hi @devongovett ! 
Do you think we can merge this and release a new version? 
I really need this 😅 thanks!

**Comment by diegomura** [comment]
Hi @devongovett !
Sorry to bother you again. Do you think we can get this merged? 
I can merge it myself, but I cannot publish to npm.
I think many people can benefit from this on the future. 
Thanks!

**Comment by dotconnor** [comment]
Any update on this?
@diegomura 
@devongovett

**Comment by devongovett** [comment]
Updated slightly to use a base64 decoding library rather than including the entire buffer polyfill. Published in v1.2.0.

**Pull request merged**